test/network.bats

```diff
@@ -140,3 +142,29 @@ function check_networking() {
 
 	check_networking
 }
+
+@test "Clean up network if pod sandbox gets killed" {
+	start_crio
+
+	CNI_RESULTS_DIR=/var/lib/cni/results
+	POD=$(crictl runp "$TESTDATA/sandbox_config.json")
+
+	# CNI result is there
+	# shellcheck disable=SC2010
+	[[ $(ls $CNI_RESULTS_DIR | grep "$POD") != "" ]]
+
+	# kill the sandbox
+	run "$CONTAINER_DEFAULT_RUNTIME" --root /var/run/runc kill "$POD"
```

Comment:
1: we can call the `runtime` function here 
2: why use `run`? we're not checking output. if it's just to prevent a failure from kiling the test, we could use `|| true`